PfMultiplayerCmdlets/GetPFMultiplayerImageTags.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.Management.Automation;
+using PlayFab.MultiplayerModels;
+
+namespace PFMultiplayerCmdlets
+{
+    [Cmdlet(VerbsCommon.Get, "PFMultiplayerContainerImageTags")]
+    [OutputType(typeof(List<string>))]
+    public class GetPfMultiplayerContainerImageTags : PageableCmdlet
+    {
+        [Parameter(Mandatory = true)]
+        public string ImageName { get; set; }
+        
+        protected override void ProcessRecord()
+        {
+            ListContainerImageTagsResponse response = CallPlayFabApi(() => Instance
```

Comment:
don't we need the All option for this one?

Reply:
By deriving from PageableCmdlet we get the All option. However, the response is not actually paged today and we will always get all tags so the option will not do anything at the moment.